### ISSUE TITLE
Windows 10 / vs13 errors after #1551

### ISSUE THREAD

**Issue by Zbizu** [otland/forgottenserver#1593]
Followed tutorial, can't compile anything after #1551 with visual studio 2013 on Windows 10.
Log: http://pastebin.com/6wjjb83g

**Comment by ninjalulz** [comment]
Duplicate of https://github.com/otland/forgottenserver/issues/1557. I suggest that you use VS 2015 for now.

**Comment by Zbizu** [comment]
Ok, closing. We need a new TFS SDK for Windows, just saying.

**Comment by ninjalulz** [comment]
You only need to update MPIR, which I uploaded [here](https://otland.net/threads/compiling-tfs-1-0-1-1-latest-source-64-and-32-bits-pics-and-lots-of-explanation.204532/page-10#post-2282390).
